numpy/_core/tests/test_scalarmath.py

```diff
@@ -875,8 +876,8 @@ def test_operator_object_left(o, op, type_):
 
 
 @given(sampled_from(objecty_things),
-       sampled_from(reasonable_operators_for_scalars),
-       sampled_from(types))
+       sampled_from(reasonable_operators_for_scalars + [operator.xor]),
```

Comment:
Does stuff break if you add xor to `reasonable_operators_for_scalars`? If not i'd just add it there. If so, maybe a comment explaining why only xor here?

Reply:
I'll think a bit if I can think of a change. I think that would be a rename of the variable and not a comment here.